sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 
+- Added new error type `IncompleteReadError` which is raised if peer closes connection without sending complete message body.
```

Comment:
Maybe we could rephrase as '...if peer closes the connection before we have received the complete message body.'
When I see 'sending', I think 'outgoing data'.